rtd/source/networking/nuage-plugin.rst

```diff
@@ -0,0 +1,249 @@
+
+3. If User Data service is desired in an Isolated Network, choose **VirtualRouter** as the User Data provider. **Per Zone** MUST be selected for the Source NAT Type for the Source NAT service.
+
+4. Click OK to create the offering. 
+
+5. After the offering has been successfully created, enable it from the Service Offerings list.
+
+Create and Enable VPC Network Offering
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. Select **Service Offerings > Select Offering**: **Network Offerings > Add network offering**.
+
+2. Select the **VPC checkbox**. In the Supported Services field, select each of the following services and then select Nuage VSP as the Provider.
+
+   *	DHCP
+   *	Source NAT
+   *  Static NAT
+   *	Virtual Networking 
+
+3. (Optional) Select **VirtualRouter** as the UserData provider if password reset or metadata feature is desired.
+
+4. (Optional) If network ACL is required, select **NuageVsp** as the network ACL provider. 
+
+   a) Ensure the *Persistent* checkbox is selected.
+   b) As the *Supported Source NAT Type*, select **Per Zone**.
+
+5.  After the offering has been successfully created, enable it from the Service Offerings list.
+
+Dedicated Features That Come with Nuage VSP Plugin
+--------------------------------------------------
+
+Domain Template Support for CloudStack in VSP
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Overview
+~~~~~~~~
+
+VSP's CloudStack plugin can be configured to use a VSD template when instantiating domains. The parameters and abstractions contained in the template are reused every time a new domain instance is created in CloudStack, and thus all the constructs defined in the template are available to the domain. 
+
+Configuration
+~~~~~~~~~~~~~
+
+Details of the global variables that have been added to support domain templates are listed below: 
+
+:nuagevsp.isolatedntwk.domaintemplate.name: (Type: string) Name of the template to use for creation of domains for isolated networks.
+
+:nuagevsp.vpc.domaintemplate.name: (Type: boolean) Name of the template to use for creation of domains for VPC.
+
+To configure a domain template for use by CloudStack, use VSD to create a domain template, using the global CloudStack parameters listed above.
+
+.. Note:: There will be only a single domain instance for ``nuagevsp.vpc.domaintemplate.name``.
+
+Networks created in CloudStack will then use domain instances created from the template to which the name points.
+
+
+
+Revision History
+----------------
+
+Fiona Fincannon 21 December 2015
+
+Appendix
+--------
+Nuage VSD API
+~~~~~~~~~~~~~
+
+To add Nuage VSP as Network Service Provider, 
+
+1.  Add the specified network service provider:
+
+::
+
+        http://135.227.147.106:8080/client/api?command=addNetworkSer
+        viceProvider&name=NuageVsp&physicalnetworkid=87528ea5-0189-4
+        a02-92db-3d1539232e21&response=json&sessionkey=CaDCr2P1qpIqm
+        Fsr%2BmMl1T3nLzs%3D&_=1414200788068
```

Comment:
That's weird.
First what is that IP for the mgt server ? it should be more generic.
Second the networkId and session key will be different ....
I suggest giving an example with a cloud monkey CLI.